source/MapPanel.cpp

```diff
@@ -1130,12 +1155,49 @@ void MapPanel::DrawSystems()
 	if(commodity == SHOW_GOVERNMENT)
 		closeGovernments.clear();
 
-	// Draw the circles for the systems.
+
 	double zoom = Zoom();
+
+	// Draw the star sprites for the systems.
 	for(const Node &node : nodes)
 	{
+		float ringOuter;
+		float ringInner;
+		int starsOn;
```

Comment:
You can remove this? It looks like you're using it as a bool to contain the same information as isStarry